src/server/services/procedures/icesheets/icesheets.js

```diff
@@ -0,0 +1,169 @@
+    .split('\n')
+    .forEach(line=>importdata3(line));
+
+/**
+ * Get delta 18O value.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} delta 18O
+ */
+Icesheets.getd180 = function(startyear, endyear) {
+    return this._data1
+        .map(data => [data.adjustTime, data.d180])
+        .filter(data => data[0] >= startyear && data[0] <= endyear);
+};
+
+/**
+ * Get delta 18O error value.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} delta 18O error
+ */
+Icesheets.getd180error = function(startyear, endyear) {
+    return this._data1
+        .map(data => [data.adjustTime, data.error])
+        .filter(data => data[0] >= startyear && data[0] <= endyear);
+};
+
+/**
+ * Get average sedimentation rate value.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} average sedimentation rate
+ */
+Icesheets.getAveSedRates = function(startyear, endyear) {
+    return this._data2
+        .map(data => [data.adjustTime, data.averagedSedRates])
+        .filter(data => data[0] >= startyear && data[0] <= endyear);
+};
+
+/**
+ * Get normalized sedimentation rate value.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} normalized sedimentation rate
+ */
+Icesheets.getNormalizedSedRates = function(startyear, endyear) {
+    return this._data2
+        .map(data => [data.adjustTime, data.normalizedSedRates])
+        .filter(data => data[0] >= startyear && data[0] <= endyear);
+};
+
+/**
+ * Get s95 time.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} s95 time
+ */
+Icesheets.gets95Time = function() {
+    return this._data3
+        .map(data => [data.s95, data.LR04a]);
+};
+
+/**
+ * Get specMap time.
+ *
+ * If a start and end year is provided, only measurements within the given range will be
+ * returned.
+ *
+ * @param {Number=} startyear Year to begin data at
+ * @param {Number=} endyear Year to begin data at
+ * @returns {Array} specMap time
+ */
+Icesheets.getsspecMapTime = function() {
+    return this._data3
+        .map(data => [data.specMap, data.LR04b]);
```

Comment:
Since we return both specMap and LR04b, would it make more sense to reference something about this in the help comment? I am honestly not sure as I am not familiar with either of these concepts :/ 

Either way, my point is that we should make sure it is clear about the meaning of the returned data. We also should make sure that the order of the columns is intuitive. For example, it could make more sense to switch the columns. (Again, I am not familiar enough with the concepts to be able to say definitively.)